code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm

```diff
@@ -143,6 +143,13 @@
 		J.species_tag = tag
 	return J
 
+/datum/gear/head/skrell_helmet
+	display_name = "Skrellmet"
+	path = /obj/item/clothing/head/helmet/security/skrell
+	whitelisted = list(SPECIES_SKRELL)
+	sort_category = "Xenowear - Skrell"
```

Comment:
Should probably allow this for HoP and Captain too since they get helmets in their lockers.